socorro/external/es/new_crash_source.py

```diff
@@ -0,0 +1,83 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import datetime
+
+from elasticsearch import helpers
+from configman import Namespace, RequiredConfig, class_converter
+
+
+class ESNewCrashSource(RequiredConfig):
+
+    required_config = Namespace()
+    required_config.add_option(
+        'cap',
+        default=0,
+        doc='If set to something other than 0, caps how many to yield'
+    )
+    required_config.elasticsearch = Namespace()
+    required_config.elasticsearch.add_option(
+        'elasticsearch_class',
+        default='socorro.external.es.connection_context.ConnectionContext',
+        from_string_converter=class_converter,
+        reference_value_from='resource.elasticsearch',
+    )
+
+    #--------------------------------------------------------------------------
```

Comment:
Yeah, I guess at some point we'll remove those all-dashes lines. Right now I mostly care about consistency inside files.